backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/invoicecandidate/C_Invoice_Candidate_StepDef.java

```diff
@@ -363,10 +363,13 @@ public void update_C_Invoice_Candidate(@NonNull final DataTable dataTable)
 	}
 
 	@And("^there is no C_Invoice_Candidate for C_Order (.*)$")
-	public void validate_no_C_Invoice_Candidate_created(@NonNull final String orderIdentifier)
+	public void validate_no_C_Invoice_Candidate_created(@NonNull final String orderIdentifier) throws InterruptedException
 	{
 		final I_C_Order order = orderTable.get(orderIdentifier);
 
+		// give IC handlers some time
+		Thread.sleep(1000);
```

Comment:
Rather than waiting a fixed amount, please use StepDefUtil.tryAndWait(maxWaitSeconds, checkingIntervalMs, worker) instead.
Waiting for a fixed amount is a recipe for either long runtimes or flaky tests, because the the processes on which you're waiting did not have enough time to do their expected job.

Reply:
As talked: `StepDefUtil.tryAndWait` doesn't work because we wait and check for something NOT created. We couldn't find a better solution, so we stay with this for now.

Note: We're waiting for IC-handlers that create missing ICs and are called from ModelInterceptors